Driverless AI/build-your-own-custom-recipe-tutorial/build-your-own-custom-recipe-tutorial.md

```diff
@@ -84,11 +86,11 @@ Once the data has been loaded to Driverless AI, Driverless AI performs Automatic
 
 The data is then sent through Driverless AI’s Automatic Model Optimization. The Automatic Model Optimization is a generic algorithm that learns over time what is working and what is not to make the best model for your data. This includes model recipes, advanced feature engineering, algorithms (such as Xgboost, TensorFlow, LightGBM), and model tuning. 
 
-After the model has been finalized, Driverless AI then auto-generates model documentation that provides an explanation of everything that happened in the experiment and how the model generated makes decisions. Additionally, there is Machine Learning Interpretability of the models generated to explain modeling results in a human-readable format. Once experiments have been completed, Driverless AI automatically generates both Python and Java scoring pipelines so that the model is ready to go for production.
+After the model has been finalized, Driverless AI then auto-generates model documentation that provides an explanation of everything that happened in the experiment and how the model generated decisions. Additionally, there is Machine Learning Interpretability of the models generated to explain modeling results in a human-readable format. Once experiments have been completed, Driverless AI automatically generates both Python and Java scoring pipelines so that the model is ready to go for production.
```

Comment:
"...how the **model generated makes** decisions" In this sentence, we are talking about the model that was generated and how it makes decisions... by removing "makes" we do not know what we are trying to say about the generated model. Maybe you can rewrite it as:
"and how the model that was _generated/created_ makes decisions.

Reply:
@FrankJVA thank you. I made sure to add in your rewrite suggestion. It will be in the new commit.